src/tlo/methods/cardio_metabolic_disorders.py

```diff
@@ -1402,6 +1403,7 @@ def apply(self, person_id, squeeze_factor):
             return hs.get_blank_appt_footprint()
 
         # Run a test to diagnose whether the person has condition:
+        self.add_equipment({'Blood pressure machine'})
         dx_result = hs.dx_manager.run_dx_test(
             dx_tests_to_run='assess_hypertension',
```

Comment:
Are there any essential consumables for the 'assess_hypertension' test? If they are, they will be checked within the run_dx_test, hence it need to be used as consumable availability check, ie equipment added only if dx_result is not None.

Reply:
Currently the assumption is that the `assess_hypertension` test is simply a check of an indivudals blood pressure using a blood pressure machine. As you can see from the definition of this dx test at line 537 there are no cons for this test.